src/gallery/event.js

```diff
@@ -0,0 +1,16 @@
+let eventListenerStack = []
```

Comment:
Should all gallery event listeners be included in this new array? The ones in `applyGestureEvent` for example? Mainly asking because that would overlap the image zoom feature I'm working on.